support_matrix/index.md

```diff
@@ -0,0 +1,885 @@
+# Support Matrix
+
+## Supported platforms and features in {{ site.data.product.title_short }}
+
+A summary of the supported platforms and features in {{ site.data.product.title_short }}.
+{:shortdesc}
```

Comment:
What does this do?

Reply:
Also on the other places...I don't think they are necessary since we generate anchors automatically

Reply:
maybe @dehawkins512 knows